testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -871,6 +887,44 @@
             </build>
         </profile>
 
+        <profile>
+            <id>map-storage-hot-rod</id>
+            <properties>
+                <keycloak.testsuite.start-hotrod-container>true</keycloak.testsuite.start-hotrod-container>
+            </properties>
+            <build>
+                <plugins>
+                </plugins>
+                <pluginManagement>
```

Comment:
Problem is with this `<pluginManagement>` section. surefire plugin should go above into <plugins> section.
After this change it will work without extra profile for `chm`. I don't have a strong opinion whether keep or remove extra profile for `chm`.